src/lib/pages/schedule-view.svelte

```diff
@@ -137,11 +140,13 @@
   const handlePause = async (schedule: DescribeScheduleResponse) => {
     schedule?.schedule?.state?.paused
       ? await unpauseSchedule({
+          identity,
```

Comment:
Can this ternary be an if statement instead it feels like it would be clearer what is happening.